Source/ACE.Server/Entity/Fellowship.cs

```diff
@@ -694,4 +730,40 @@ public void ProcessDropList(Dictionary<uint, WeakReference<Player>> fellowshipMe
             UpdateAllMembers();
         }
     }
+
+    public class DepartedMembersComparer : IComparer<uint>
+    {
+        public static ushort TableSize = 32;
```

Comment:
are you sure the table size differed here from the main fellowships table? it seems odd that one would use 16, and another would use 32

if they indeed differed, this should be implemented as a constructor that takes the table size. This logic shouldn't be duplicated for ever new table size

Reply:
i have no idea